src/scriptlets/prevent-refresh.js

```diff
@@ -71,6 +71,9 @@ export function preventRefresh(source, delaySec) {
                 return contentDelay;
             })
             .filter((delay) => delay !== null);
+        // Check if "delays" array is empty, may happens when meta's content is invalid
+        // and reduce() method cannot be used with empty arrays without initial value
+        if (!delays.length) { return; }
```

Comment:
we do not use one-line return statement, better
```
if (!delays.length) {
    return;
}
```

Reply:
OR 
you can obviously return `null` here and check it by strict equality after it
https://github.com/AdguardTeam/Scriptlets/pull/238/files#diff-4b774935780fa37cb477fd62f4db8198f3d8d0447b0fc2824e1b7bb13d349201R94

Reply:
Okay, I have changed it as requested.